zion/src/test/scala/edu/uci/ics/cloudberry/zion/actor/DataStoreManagerTest.scala

```diff
@@ -291,13 +281,80 @@ class DataStoreManagerTest extends TestkitExample with SpecificationLike with Mo
       sender.expectMsg(DataManagerResponse(false, "Register Denied. Field Parsing Error: " + "Time field of " + schemaNotATimeField.typeName + "is not in TimeField format.\n"))
       ok
     }
-    "respond success if deregister an existing data model" in {
+  }
+
+  "Data schema deregistering process" should {
+    val sender = new TestProbe(system)
+    val view = new TestProbe(system)
+    val base = new TestProbe(system)
```

Comment:
My question is that since the actors are created when `answerQuery` is called, what we need to do is to send two queries and then drop dataset. In this case `PoisonPill` for `base` and `view` could be caught, right? Rather than loading a dataset and its view from initialization.